INSTALL.md

```diff
@@ -1,17 +1,55 @@
-# Install Zync (only explained for MacOS so far)
-1. Download this repository.
+# Install Zync
+
+## Download this repository.
 `git clone git@github.com:3scale/zync.git`
 
-2. Move to the folder of the project.
-`cd zync`
+## Setup PostgreSQL on Mac
+
+There is `Brewfile` containing all the dependencies.
+
+```
+cd zync
+brew bundle
+brew services start postgresql
+```
+
+The command `brew services start postgresql` starts the service of PostgreSQL. If later `./bin/setup` aborts, make sure that the `PostgreSQL` service is running. Verify with `brew services list` that has a status `started` and looking green. If the status is `started` but coloured orange, fix the errors indicated in the log located in `/usr/local/var/log/postgres.log`
+
+## Setup PostgreSQL on Fedora 34
 
-3. Install the dependencies. There is `Brewfile` containing all the dependencies.
-`brew bundle`
+```shell
+sudo dnf module install postgresql:10
+sudo dnf install libpq-devel
+sudo /usr/bin/postgresql-setup --initdb
+systemctl enable postgresql
+systemctl start postgresql
+psql -U postgres -c "create user ${USER}"
+psql -U postgres -c "create database zync_development with owner ${USER}"
+psql -U postgres -c "create database zync_test with owner ${USER}"
+psql -U postgres -c "create database zync_production with owner ${USER}"
```

Comment:
You do not need this... Postgresql creation of user and db is known/documented by them and standard
https://www.postgresql.org/docs/current/app-createdb.html

I really do not think we should document other 3rd party software ... it looks like Postgresql guide more than zync ... and maintaining it will be tedious. 

I would prefer you just put a sentence:

> Setup a postgresql role with `createdb` and `login` grants

I really prefer assuming people know at least the basic of postgresql than documenting all for them because otherwise we will have to teach them rails, sql etc ... 

Hint how to do it on Fedora (no need to document here): 

```
sudo su - postgres
createuser <your_username> --createdb --login
exit
```

Then `./bin/setup` will create all the things for you

Reply:
I don't have executable `createdb` on my Fedora. I just put here the commands that worked for me. Same about copy/paste instructions like on the other two places applies here.

Reply:
You need to login with `postgres` user to have it ;)